Mathlib/Computability/Partrec.lean

```diff
@@ -546,8 +546,8 @@ theorem bind_decode_iff {f : α → β → Option σ} :
             snd).bind
         (Computable.comp hf fst).to₂.partrec₂)
       fun n => by
-        simp; cases decode (α := α) n.unpair.1 <;> simp;
-          cases decode (α := β) n.unpair.2 <;> simp,
+        simp; cases decode (α := α) n.unpair.1 <;> simp
```

Comment:
could clear up another semicolon here at the same time:
```suggestion
        simp
        cases decode (α := α) n.unpair.1 <;> simp
```

Reply:
(which is to say; I think this is deliberately a long wrapped line of semicolons, and the fix is to remove all the semicolons, not just the one at EOL)

Reply:
Good point, fixed!